src/lib/Components/Consignments/Screens/Overview.tsx

```diff
@@ -118,18 +120,39 @@ export default class Info extends React.Component<Props, State> {
       updateSubmission(this.state, this.state.submission_id)
     } else if (this.state.artist) {
       const submission = await createSubmission(this.state)
-      this.setState({ submission_id: submission.id })
+      this.setState({ submission_id: submission.id }, this.submissionDraftCreated)
     }
   }
 
+  @track((props, state) => ({
+    action_type: Schema.ActionTypes.Success,
+    action_name: Schema.ActionNames.ConsignmentDraftCreated,
+    owner_id: state.submission_id,
+    owner_type: Schema.OwnerEntityTypes.Consignment,
+    owner_slug: state.submission_id,
+  }))
+  // tslint:disable-next-line:no-empty
+  submissionDraftCreated() {}
```

Comment:
hrm, this function is being called, but the analytics aren't being sent

Reply:
what if you add `null` to the func body?

Reply:
When you say that the analytics aren’t being sent, do you mean the data collection function isn’t called?

Reply:
If not, I wonder if it has anything to do with the comment in between?

Reply:
Will try both ( and add a note in that notes section) this one stumped me, I have a test of  for it

Reply:
Looks like it was because I was using `@track` but I needed to use `@pageViewTrack` - there must be a difference in terms of "this one is used at the root, and this one is used for a component lower down the tree"